src/seedu/addressbook/ui/TextUi.java

```diff
@@ -132,6 +132,14 @@ public void showResultToUser(CommandResult result) {
         showToUser(result.feedbackToUser, DIVIDER);
     }
 
+    public void showListAllResultToUser(CommandResult result) {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.